util/feature.txt

```diff
@@ -0,0 +1,5 @@
+feature 1 implementation
+
+feature 2 implementation
```

Comment:
let's add a bit more to the feature 2 implementation. I think it should handle multiple widgets

Reply:
@chriseboy see the latest commit for implementation